JitsiMeetJS.ts

```diff
@@ -244,6 +251,13 @@ export default {
      * @see Logger.addGlobalTransport
      */
     addGlobalLogTransport(globalTransport) {
```

Comment:
not sure if this is used by ljm customers, but if they did that would mean that they'd disable sending stats to rtcstats right?

Reply:
Yep, if they don't send them, yeah. 
Like jitsi-meet is sendgin them:
https://github.com/jitsi/jitsi-meet/blob/59420ea1a8204201db0b3203818594ef4ac66462/react/features/base/logging/JitsiMeetLogStorage.ts#L75

Reply:
I wonder if it's common for them to use it, because if it is then it probably defeats the purpose. cause we still have to tell them to add sending logic in their app.

Reply:
Well then you have a custom one and handling the stuff yourself and do not need the default one ... that is the idea ... like jitsi-meet pushes one and we use that one ... 
I don't have a better idea how to handle that in another way.

Reply:
Can't you have 2 loggers? so basically make ljm always send to rtcstats unless disabled by the config?

Reply:
But then, how to handle that with jitsi-meet without having duplicate logs?

Reply:
Yeah, I'm confused ... I will take another look at it today.

Reply:
Maybe we can disable it from jitsi meet,  and only send logs to rtcstats through ljm rtcstats logger?

Reply:
I have an  idea.

Reply:
I will do the change in a bit.

Reply:
> Maybe we can disable it from jitsi meet, and only send logs to rtcstats through ljm rtcstats logger?

And you already came to the same conclusion before me :) I missed your comment while writing mine, sorry :)

Reply:
@andrei-gavrilescu Done. 
https://github.com/jitsi/jitsi-meet/pull/15763

Once we merge this one, I will update the jitsi-meet PR with the correct ljm version.

Reply:
Looks good!